wallet/src/actors/worker/methods.rs

```diff
@@ -621,12 +641,24 @@ impl Worker {
             log::debug!("[SU] Received chain: {:?}", block_chain);
 
             // For each of the blocks we have been informed about, ask a Witnet node for its contents
-            for ChainEntry(_epoch, id) in block_chain {
+            for ChainEntry(epoch, id) in block_chain {
                 let get_block_future = self.get_block(id.clone());
                 let block: types::ChainBlock = futures03::executor::block_on(get_block_future)?;
 
-                // Process each block
-                self.handle_block(block.clone(), wallet.clone(), DynamicSink::default())?;
+                // Compute if block should be considered confirmed
+                // Note: blocks confirmed in past superblocks are considered confirmed
+                // FIXME(#1437): best approach would be the node signaling if blocks are confirmed
+                let tip_superblock_index = self.current_epoch()? / superblock_period;
+                let current_superblock_index = epoch / superblock_period;
```

Comment:
This is a bit missleading, as current refers to the block we are processing inside this loop while tip refers to the current epoch divided with the period. I would change them for:

let current_superblock_index = self.current_epoch()? / superblock_period;
let processed_superblock_index = epoch / superblock_period

Or something similar

Reply:
What about this?
```rust
                let node_tip_superblock_index = self.current_epoch()? / superblock_period;
                let local_tip_superblock_index = epoch / superblock_period;
```

Reply:
this is cool